sensu/plugins/check-l3-router.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+HOSTNAME=`hostname`
+for IFACE in $(ifquery --list); do
```

Comment:
As far as I can tell, we really only care about the case where the public VIP moves. I would change this script to accept an input parameter that is the name of the public interface.  Doing so would remove all of this finding-the-interface logic.